Assets/Scripts/EnemySword.cs

```diff
@@ -18,6 +18,11 @@ void Start()
         rigidbody = GetComponent<Rigidbody2D>();
         target = GameObject.Find("Player").GetComponent<Transform>();
         targetDirection = (target.transform.position - transform.position).normalized * speed;
+        Vector3 relativePos = target.transform.position - gameObject.transform.position;
+        Quaternion rotation = Quaternion.LookRotation(relativePos);
+        rotation.x = gameObject.transform.rotation.x;
+        rotation.y = gameObject.transform.rotation.y;
+        gameObject.transform.rotation = rotation;
         rigidbody.velocity = new Vector2(targetDirection.x, targetDirection.y);
```

Comment:
Feel like your stating properties for no reason when they can just be put into one line. Stuff like this should be in the update method if the properties are changing, otherwise save lines by putting into one line.